src/ray/gcs/format/gcs.fbs

```diff
@@ -99,6 +99,16 @@ table TaskTableTestAndUpdate {
 table ClassTableData {
 }
 
+enum ActorState:int {
+  // Actor is alive.
+  ALIVE = 0,
+  // Actor is ready dead and won't be reconstructed.
```

Comment:
I don't think "ready" is the right word.

Reply:
typo, I meant 'already'